app/views/observations/show/_observation.html.erb

```diff
@@ -32,44 +36,43 @@
   %>
 </p>
 
-<p id="observation_who">
+<p class="obs-who" id="observation_who_<%= obs_id %>">
   <%= :WHO.t %>: <span><%= user_link(observation.user) %></span>
 </p>
 
 <% if @user %>
-  <div id="observation_projects">
+  <div class="obs-projects" id="observation_projects_<%= obs_id %>">
     <% observation.projects.each do |project| %>
-          <p><%= :PROJECT.t %>: <%= link_to_object(project) %></p>
+      <p><%= :PROJECT.t %>: <%= link_to_object(project) %></p>
     <% end %>
   </div>
 <% end %>
 
-<div id="observation_specimen_available">
+<div class="obs-specimen" id="observation_specimen_available_<%= obs_id %>">
   <%= observation.specimen ? :show_observation_specimen_available.t :
                               :show_observation_specimen_not_available.t %>
 </div>
 
 <% if @user %>
-  <div id="observation_collection_numbers">
+  <div class="obs-collection" id="observation_collection_numbers_<%= obs_id %>">
     <%= render(partial: "observations/show/collection_numbers",
                 locals: { observation: observation }) %>
   </div>
 
-  <div id="observation_herbarium_records">
+  <div class="obs-herbarium" id="observation_herbarium_records_<%= obs_id %>">
     <%= render(partial: "observations/show/herbarium_records",
                 locals: { observation: observation }) %>
   </div>
 
-  <div id="observation_sequences">
+  <div class="obs-sequence" id="observation_sequences_<%= obs_id %>">
     <%= render(partial: "observations/show/sequences",
                 locals: { observation: observation }) %>
   </div>
 <% end %>
 
-<div class="obs-notes" id="observation_notes">
+<div class="obs-notes" id="observation_notes_<%= obs_id %>">
   <%= if observation.notes?
```

Comment:
More specific IDs necessary when this partial appears with other versions of the same partial on the page (in lightboxes). The IDs are not unique in that case.